File: 4A.c
#include<stdio.h>
int main (){
    int w;
    scanf("%d",&w);
    if(w>2){
        if(w%2==0){
            printf("YES\n");
        }
        else{
            printf("NO\n");
        }
    }
    else{
        printf("NO\n");
    }
    return 0;
}

File: 4A.java
import java.util.Scanner;
public class 4A{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int w = sc.nextInt();
        if (w%2==0 && w!=2){
            System.out.println("YES");
        }
        else{
            System.out.println("NO");
        }
    }
}